01-Login/README.md

```diff
@@ -1,18 +1,21 @@
-# Express OpenID Connect sample
+# Express OpenID Connect Webapp Sample
 
-See a detailed walk-through of this app on the [Express Quickstart](https://auth0.com/docs/quickstart/webapp/express).
+See a detailed walk-through of this sample app on the [Express Quickstart](https://auth0.com/docs/quickstart/webapp/express).
 
-## Running the Sample
+## Running This Sample Locally
 
-If you don't yet have an Auth0 account, [sign up](https://auth0.com/signup) for free.
+> If you don't yet have an Auth0 account, you can [sign up](https://auth0.com/signup) for free.
```

Comment:
I'm not sure this one makes sense as a blockquote, and when I think about it, the wording on this is also kind of awkward

Reply:
I'm with you on the "awkward" aspect. And better ideas making this a side-note instead of primary journey content? Then again, maybe what I was thinking doesn't matter.